### ISSUE TITLE
Stop seeding the random number generator

### ISSUE THREAD

**Issue by rj-jesus** [MartinThoma/HASY#17]
Hi,

In my opinion the [package](https://pypi.org/project/hasy-tools/) should not seed any random number generators (in this case python's and numpy's). Otherwise the usage of the package becomes quite limited since its users have to worry about setting them themselves, or worse, not even realize that they are not actually generating random test runs.

If you want to keep its effect maybe move the related statements to the `__main__` region.

Best regards,
Ricardo Jesus

**Comment by MartinThoma** [comment]
Good point! When I wrote this package I had way less experience with package building. I will change it (hopefully this weekend)
